src/observable/ConnectableObservable.ts

```diff
@@ -39,6 +39,15 @@ export class ConnectableObservable<T> extends Observable<T> {
   refCount(): Observable<T> {
     return new RefCountObservable(this);
   }
+
+  /**
+   * This method is opened for `ConnectableSubscription`.
+   * Not to call from others.
+   */
```

Comment:
Rather than this comment, I'd just name it `_closeSubscription` ... generally, `_name` is idiomatic in the JavaScript community for "this is not a public API".